test/invalidResponse.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const assert = require('assert');
+const proxyquire = require('proxyquire');
+
+const Z = require('../');
+const mockHttpClient = require('./helpers').mockHttpClient;
+
+
+describe('requests()', () => {
```

Comment:
Why `requests()` when there is no(?) function called `requests` anywhere? I would just call this test spec `requests` or something. No big deal though! 👍

Reply:
Ah, I have no idea why I named it like that, probably just entered something and then forgot to fix it before committing.

I don't think that this needs a new release if you haven't encountered it until now. Under normal circumstances everything should work, but when I for example broke my dev environment this made it harder to figure out what was going on.

Reply:
Yeah, that's a very valid point and I'm glad you fixed it. But lets hold off on the release. In the meantime, maybe you can introduce this change locally and let it sit there while you work on other things, just to confirm that it doesn't break anything (which it shouldn't – a `reject()` is a `reject()` is a `reject()`).